tests/UnifiedSpecTests/Util.php

```diff
@@ -33,6 +34,19 @@ public static function assertHasOnlyKeys($arrayOrObject, array $keys): void
         assertEmpty($diff, 'Unsupported keys: ' . implode(',', array_keys($diff)));
     }
 
+    /**
+     * @param string $executingObjectName
+     * @param string $operation
+     * @param array  $args
+     * @return void
+     */
+    public static function assertParamsBySchema(string $executingObjectName, string $operation, array $args): void
+    {
+        if (isset(MethodsParams::$args[$executingObjectName][$operation])) {
```

Comment:
I realize the `isset()` helps since the `$args` array is incomplete, but ultimately I think this should be replaced with an assertion(s) that we have a list of parameter names for the operation:

 * `$args` has the `$executingObjectName` key
 * `$args[$executingObjectName]` is an array
 *  `$args[$executingObjectName]` has the `$operation` key
 * `$args[$executingObjectName][$operation]` is an array

Alternatively, you can use the [null coalescing operator](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce). A single assertion that `$args[$executingObjectName][$operation] ?? null` is an array should have the same effect, and be more concise/performant.